core/modules/inputmanager.js

```diff
@@ -0,0 +1,72 @@
+/*\
+title: $:/core/modules/inputmanager.js
+type: application/javascript
+module-type: global
+
+Input handling utilities
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
```

Comment:
Comment block with general info is missing.

Reply:
IMO there should be a global variable, that contains the tiddler name. It makes it easier to find typos.

```
var STATE_CURRENT_FOCUS = "$:/state/current-focus"
```

it's used in comments below.